engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -95,7 +95,9 @@
 public class StramClient
 {
   private static final Logger LOG = LoggerFactory.getLogger(StramClient.class);
-  public static final String YARN_APPLICATION_TYPE = "DataTorrent";
+  public static final String YARN_APPLICATION_TYPE_DEPRECATED = "DataTorrent";
+  public static final String YARN_APPLICATION_TYPE = "ApacheApex";
```

Comment:
Apologies if this feedback is any late, but I suggest not to attach any organization's name to yarn application type or any such string. Reason being same as why we have to change it now.
Suggesting to make it "Apex" OR "ApexStreaming"